docs/source/conf.py

```diff
@@ -11,14 +11,10 @@
 from datetime import datetime
 
 
-sys.path.insert(0, os.path.abspath('../..'))
-sys.path.insert(0, os.path.abspath('../../autometa/common'))
-sys.path.insert(0, os.path.abspath('../../autometa/config'))
-sys.path.insert(0, os.path.abspath('../../autometa/binning'))
-sys.path.insert(0, os.path.abspath('../../autometa/datasets'))
-sys.path.insert(0, os.path.abspath('../../autometa/validation'))
-sys.path.insert(0, os.path.abspath('../../autometa/taxonomy'))
-sys.path.insert(0, os.path.abspath('../../autometa/common/external'))
+sys.path.append(os.path.abspath("./_ext"))
+
+for (dir_path,dir_names,file_name) in os.walk('../../', topdown=True): 
+    sys.path.insert(0, os.path.abspath(dir_path))
```

Comment:
Nice! great way to reduce 8 lines to 2 _and_ no more having to add new lines for each module! 👍